src/Illuminate/Validation/Validator.php

```diff
@@ -175,14 +175,14 @@ class Validator implements ValidatorContract
      *
      * @var array
      */
-    protected $sizeRules = ['Size', 'Between', 'Min', 'Max'];
+    protected $sizeRules = ['Size', 'Between', 'Min', 'Max', 'greater_than', 'less_than', 'greater_than_or_equal', 'less_than_or_equal'];
```

Comment:
Is this right again? Because it is `Size` (notice the upper case first letter) - but you have `greater_than`.

Shouldnt it be `GreaterThan` etc?

Reply:
You're right again, sorry it was 4 am here and i was sleepy :D